posts/2021-08-30-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc

```diff
@@ -0,0 +1,284 @@
+---
+layout: post
+title: "Bind an Operator Managed PostgreSQL Database to a JPA Application in a Kubernetes Cluster"
```

Comment:
here and throughout, use sentence-style capitalization for title, not headline style (and hyphenate "operator-managed"...)

Bind an operator-managed PostgreSQL database to a JPA application in a Kubernetes cluster